scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -689,11 +689,11 @@
       </div>
       <div class="col-2">
         {{ form.tumor_frequency.label(class="control-label") }}
-        {{ form.tumor_frequency(class="form-control") }}
+        {{ form.tumor_frequency(class="form-control", step="0.01", min="0", max=1) }}
```

Comment:
These are not really in play - they belong to `Decimal()`, which we may or may not want to switch back to.